pkg/kubelet/kubelet_volumes.go

```diff
@@ -128,7 +130,13 @@ func (kl *Kubelet) removeOrphanedPodVolumeDirs(uid types.UID) []error {
 	}
 	if len(volumePaths) > 0 {
 		for _, volumePath := range volumePaths {
-			if err := syscall.Rmdir(volumePath); err != nil {
+			var err error
+			if strings.Contains(volumePath, emptydir.GetQualifiedName()) {
+				err = os.RemoveAll(volumePath)
```

Comment:
Function description should be updated. It states that "t effectively does a recursive rmdir instead of RemoveAll", which is not the case anymore.

Reply:
Please bear in mind that using `RemoveAll` instead of `Rmdir` might slow down the execution which is part of the kubelet's main sync loop iteration. I'm not saying it's wrong, just that IIUC this code was written with an assumption that no files exist anymore. We should be careful here.